composer.json

```diff
@@ -26,7 +26,7 @@
     },
     "require-dev": {
         "codeigniter/framework": "~3.0",
-        "illuminate/support": "^5.4",
+        "illuminate/support": "^7.0",
```

Comment:
Don't update exclusively the dependency's major version. It can be a possible project-breaker for some users.